test/e2e/network/network_policy.go

```diff
@@ -351,6 +380,85 @@ var _ = SIGDescribe("NetworkPolicy", func() {
 			testCanConnect(f, nsB, "client-b", service, 80)
 		})
 
+		ginkgo.It("should enforce policy to allow traffic only from a pod in a different namespaces based on PodSelector and NamespaceSelector [Feature:NetworkPolicy]", func() {
```

Comment:
"in a different namespace" (not "namespaces")

Reply:
Updated "namespaces" to "namespace".